backend/certificates.js

```diff
@@ -79,26 +79,30 @@ export function fetchCertsFromCA() {
         ));
     }).then((res) => {
         if (res.status !== 201) {
-            let errorMessage = 'An error occured requesting certificates from the CA, ' +
-                `status ${res.status} was returned.`;
-            res.json().then(json => {
-                errorMessage += `JSON response was: ${JSON.stringify(json)}`;
-                Logger.log(errorMessage, 0);
-                return errorMessage;
-            }).catch(() => {
-                throw errorMessage;
-            }).then(() => {
-                throw errorMessage;
+            return res.json().then(json => {
```

Comment:
if the server's response isn't `application/json`, then `res.json()` will fail. This could happen if the server went down or the server's proxy returns its own HTML error page.

Reply:
that error would be caught by this [catch](https://github.com/n-riesco/plotly-database-connector/blob/6da27fd64a76823df2097f11dd2890dfefa8bd52/backend/certificates.js#L93)